wellcometrust/tap-salesforce
# Allow use of Bulk API 2.0

# Description of change

Adds a new class BulkV2 and a new api_type option `BULK_V2` to configure the tap to use Saleforce Bulk API 2.0.

# QA steps

Tested the el_salesforce DAG locally, which successfully ran using the V2 API. Checked a sample of output files locally to check the number of records was the same when run with v1 vs v2.
 
# Risks

# Rollback steps
 - revert this branch
